### ISSUE TITLE
Can't create Admin user

### ISSUE THREAD

**Issue by BenediktBertsch** [overleaf/overleaf#647]
## Problem

Went to http://overleaf/launchpad fill the whole data and get the successfully registered message. Goto login get the message "Your email or password is incorrect. Please try again". Request a new password for the registered email, get the message that that mail is not registered.
So launchpad doesnt work, as in the documentation described i try to make an account by terminal and get this:
![image](https://user-images.githubusercontent.com/38132773/66718721-32fa3f00-ede7-11e9-93ab-5d5aa6710710.png)

So basically i get a token to create a password for my account, but its always expired because the docker is running in a different timezone.
## Install type

Running in docker.

## Version

The latest version on hub.docker.com sharelatex/sharelatex.

## Logs and errors

Didnt find any errors in logs in mongodb, redis and overleaf.

Any idea how to get an admin user?

Kind regards.

**Comment by mserranom** [comment]
Thanks for reporting @BenediktBertsch ,

The timezone looks correct, provided you're in Germany/Austria (guessed from your keyboard config). What you see in the docker container is UTC time, which is a 2 hour difference.

I'm not able to reproduce the issue. Maybe you can share the log message from your `sharelatex` container? (`/var/log/sharelatex/web.log`)

**Comment by schneefux** [comment]
I had this issue as well, my browser and my database are in +02:00 and the container is in UTC.
When I submitted the password form, the POST would always return 404 not found (/user/set/password I think).

I worked around the problem by installing v1, creating the user and upgrading to v2.

**Comment by BenediktBertsch** [comment]
Hey @mserranom,

thank you for the fast answer! 
I checked the log and found some redis error in the realtime log, no other things...
The Docker container seems to not take the hostname of my redis i gave him by environment variables?
I will append the log. Maybe you find more information, thank you.
[real-time.log](https://github.com/overleaf/overleaf/files/3724151/real-time.log)
[web.log](https://github.com/overleaf/overleaf/files/3724152/web.log)

**Comment by mserranom** [comment]
@BenediktBertsch you might be missing environment variables, check https://github.com/overleaf/overleaf/wiki/Release-Notes-2.0

**Comment by BenediktBertsch** [comment]
@mserranom okay i added the redis_host variable and now is the real-time log without errors, but it still doesnt fix my problem... New Logs attached.
[web.log](https://github.com/overleaf/overleaf/files/3725163/web.log)
[real-time.log](https://github.com/overleaf/overleaf/files/3725165/real-time.log)

**Comment by mserranom** [comment]
@BenediktBertsch I'm afraid I can't reproduce the problem.

> basically i get a token to create a password for my account, but its always expired

The logs don't show any expiration problem, can you describe how do you get the expiration feedback?

**Comment by BenediktBertsch** [comment]
Hey @mserranom, I tried by going to overleaf/launchpad but also tried by running the $ docker exec sharelatex /bin/bash -c "cd /var/www/sharelatex; grunt user:create-admin --email joe@example.com" command, but like in the first pointed out it didnt help to create an Adminaccount because the token was always already expired because of the timezone difference i think.

Could overleaf create an environment variable for timezone, probably this would fix it? Are there no more logs which could point to any configuration problems?

**Comment by mserranom** [comment]
Managed to reproduce, will be fixed in the next release.
